### PR TITLE
Migration to the Pyramid web framework.

### PR DESCRIPTION
**Pull request by karenc**
Clean up commits to follow in another PR.

### CODE REVIEW

**Comment by reedstrm** [comment]
is this in flight, or ready for review?

**Comment by karenc** [comment]
It is ready for review

**Comment by pumazi** [comment]
I think we need to think about how this will affect deployment before we merge it. I only say this because production is far enough from master as it is. Adding this on top of it could be too much.

**Comment by reedstrm** [comment]
Yay for devops thinking @pumazi. Agreed.

**Comment by karenc** [comment]
Let's think about how to get production to current master.

**Comment by reedstrm** [comment]
This has passed checks, I think we should deploy it to dev and smoke out any issues.

**Comment by pumazi** [comment]
We may need to address the change in #355. It's something to look into anyway.

File: cnxarchive/__init__.py [hunk 1]
```diff
@@ -33,114 +28,45 @@
     'content-type',
     ]
 
+# XXX (25-Jul-2015) Should be removed ASAP
 def get_settings():
```
Comment (pumazi) [inline cnxarchive/__init__.py]: Let's add an issue for this. It's one of those things that shouldn't be left hanging around.

**Comment by pumazi** [comment]
:+1: We should add issues to remove the settings getter/setter functions and the application helpers. I can probably take care of these later this week or over the weekend.

I think we are ready to part the Red Sea. 

@reedstrm Feel free to merge it whenever you're ready. Thanks for letting me look over it first. :checkered_flag:

**Pull request merged**